mlnlffigen/sources.mlb

```diff
@@ -52,8 +52,10 @@ local
    local open basis_lib ckit_lib in
       ast-to-spec.sml
       hash.sml
+      structure NLFFIHash = Hash
```

Comment:
That's ok.  What I was suggesting was that earlier in the file, changing
```
basis mlton_lib = bas ../lib/mlton/sources.mlb end
```
to
```
basis mlton_lib = bas local ../lib/mlton/sources.mlb in structure Control structure Popt structure Process structure Promise end end
```
(for what ever structures from the MLton utility library are actually used by `mlnlffigen`).

Reply:
Yeah, I understood that, although I wasn't quite sure which structures were used or how to get them from the def use files. I figured, if it's a bit weird, that this solution is simpler/scales more easily in the end. Would you prefer me to switch it to the solution you have?

Reply:
I'd prefer to do the export filter on the MLton library.  The intention is to leave the original source code structure as unchanged as possible.